kolibri/core/discovery/utils/network/search.py

```diff
@@ -140,27 +149,30 @@ def add_service(self, zeroconf, type, name):
                 )
 
                 logger.info(
-                    "Kolibri instance '%s' joined zeroconf network; service info: %s\n"
+                    "Kolibri instance '%s' joined zeroconf network; service info: %s"
                     % (id, self.instances[id])
                 )
 
             except ValidationError:
                 import traceback
 
                 logger.warn(
-                    (
-                        "A new Kolibri instance '%s' was seen on the zeroconf network, "
-                        + "but we had trouble getting the information we needed about it.  This probably isn't a big deal!\n\n"
-                        + "service info:\n %s;\n\nThe following exception was raised:\n%s"
-                    )
+                    """
+                        A new Kolibri instance '%s' was seen on the zeroconf network,
+                        but we had trouble getting the information we needed about it.
+                        Service info:
+                        %s
+                        The following exception was raised:
+                        %s
+                        """
```

Comment:
maybe a good opportunity to switch to new-style `.format` string formatting? (pet peeve of mine, I've always hated `printf`)

Reply:
Can do more cleanup in the future - yes I dislike it too.